style.css

```diff
@@ -56,21 +57,27 @@ nav {
 }
 
 .first-btn button {
-  background-color: #1a2236;
+  background-color: #fff;
   width: 310px;
   height: 60px;
   padding: 10px;
-  color: white;
+  color: #1a2236;
   font-size: 24px;
   font-family: 'Poppins', sans-serif;
   font-weight: 500;
   border: none;
+  border: #1a2236 solid 1px;
 }
 
 button:hover {
-  color: #1a2236;
-  border: #ff7a59 solid 1px;
-  background: #fff;
+  background-color: #1a2236;
+  color: white;
+  border: #fff solid 1px;
+}
+
+button:active {
+  background-color: #1a2236;
+  border: white solid 2px;
 }
```

Comment:
- [ ] [OPTIONAL] I noticed you added the hover and active states to the `Start Collaboration` button as stated by the previous reviewer but maybe you did not understand it well, but this needs to be added for all buttons on the page. I am only adding this as optional because It might not have been very clear.